src/app/containers/InlineLink/__snapshots__/index.test.jsx.snap

```diff
@@ -1,6 +1,152 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`InlineLinkContainer with link matching routes for SPA should render correctly 1`] = `
+exports[`InlineLinkContainer external link accessibility should be explicitly marked "external" for screen reader users & localised 1`] = `
+<Context.Provider
+  value={
+    Object {
+      "articleAuthor": "https://www.facebook.com/bbcnews",
+      "brandName": "BBC News فارسی",
+      "defaultImage": "https://news.files.bbci.co.uk/ws/img/logos/og/persian.png",
+      "defaultImageAltText": "BBC News فارسی",
+      "externalLinkText": "، لینک خارجی",
+      "imageCaptionOffscreenText": " ، عنوان تصویر",
+      "imageCopyrightOffscreenText": " ، منبع تصویر",
+      "locale": "fa",
+      "service": "persian",
+      "serviceName": "Persian",
+      "themeColor": "#B80000",
+      "translations": Object {
+        "error": Object {
+          "404": Object {
+            "callToActionFirst": "روش دیگر، لطفآ به ",
+            "callToActionLast": " بازدید کنید",
+            "callToActionLinkText": "صفحه اصلی بی بی سی فارسی",
+            "callToActionLinkUrl": "https://www.bbc.com/persian",
+            "message": "متاسفانه به دلیلی این صفحه در دسترس نمی‌باشد. لطفآ این رو امتحان کنید:",
+            "solutions": Array [
+              "لینک را چک کنید",
+              "صفحه را ریفرش کنید",
+              "با استفاده از نوار جستجوی بی بی سی برای این صفحه جستجو کنید",
+            ],
+            "statusCode": "۴۰۴",
+            "title": "این صفحه یافت نشد",
+          },
+          "500": Object {
+            "callToActionFirst": "روش دیگر، لطفآ به ",
+            "callToActionLast": " بازدید کنید",
+            "callToActionLinkText": "صفحه اصلی بی بی سی فارسی",
+            "callToActionLinkUrl": "https://www.bbc.com/persian",
+            "message": "متاسفانه به دلیلی این صفحه در دسترس نمی‌باشد. لطفآ این رو امتحان کنید:",
+            "solutions": Array [
+              "صفحه را ریفرش کنید",
+              "بعدا برگردید",
+            ],
+            "statusCode": "۵۰۰",
+            "title": "مشکلی پیش آمده است",
+          },
+        },
+      },
+      "twitterCreator": "@bbcpersian",
+      "twitterSite": "@bbcpersian",
+    }
+  }
+>
+  <InlineLinkContainer
+    blocks={
+      Array [
+        Object {
+          "model": Object {
+            "attributes": Array [
+              "",
+            ],
+            "text": "این لینک هست",
+          },
+          "type": "fragment",
+        },
+      ]
+    }
+    isExternal={true}
+    locator="https://www.example.com/"
+  />
+</Context.Provider>
+`;
+
+exports[`InlineLinkContainer external link accessibility should be explicitly marked "external" for screen reader users 1`] = `
```

Comment:
I dont think this snapshot is testing what we want, might need to do a deep render